builtin/logical/pki/crl_util.go

```diff
@@ -30,6 +38,18 @@ type revocationInfo struct {
 	CertificateIssuer issuerID  `json:"issuer_id"`
 }
 
+type (
+	deltaWALInfo struct{}
+	lastWALInfo  struct {
+		// Info to write about the last WAL entry.
+		Serial string `json:"serial"`
```

Comment:
I'm not clear if this is 
a) the serial number of the last certificate revoked, added to the WAL
b) the serial number of the last CRL referenced by the DeltaCRL
c) the serial number of the last deltaCRL.

I'm not sure if this needs to be fixed here, but a comment would be nice (trying to figure out what the last WAL entry is is not easy)